prep-css-layout-classes/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CSS Layout Classes</title>
+  <link rel="stylesheet" href="reset.css">
+  <style>
+    /* stylelint-disable block-no-empty */
+    body {
+      font-family: sans-serif;
+    }
+
+    h1 {
+      text-align: center;
+    }
+
+    img {
+      width: 100%;
+    }
+    .container {
+      max-width: 992px;
+      margin: auto;
+
+    }
+
+    .row {
+      width: 100%;
+      display: flex;
+
+    }
+
+    .column {
+      flex-basis: 100%/3
+      padding left: 8px;
+      padding right: 8 px;
```

Comment:
to get these properties to work they'll need some corrections.

```js
flex-basis: calc(100%/3);
padding-left: 8px;
padding-right: 8px;
```

Reply:
<img width="1554" alt="Screen Shot 2022-09-26 at 12 05 31 PM" src="https://user-images.githubusercontent.com/113474457/192359263-fc955eed-96bc-4486-9061-61f37c65379d.png">